packages/vite/src/node/config.ts

```diff
@@ -1093,6 +1093,14 @@ async function loadConfigFromBundledFile(
   // with --experimental-loader themselves, we have to do a hack here:
   // write it to disk, load it with native Node ESM, then delete the file.
   if (isESM) {
+    // check and clear legacy Tmp files
+    const fileBaseName = path.basename(fileName)
+    const fileDir = path.dirname(fileName)
+    const timestampFiles = (await fsp.readdir(fileDir)).filter(name=>name.includes(`${fileBaseName}.timestamp-`))
+    timestampFiles.forEach(name=>{
+      fs.unlink(path.resolve(fileDir,name), () => { });
+    })
```

Comment:
Two vite processes may be executed using different cache dirs on the same folder at the same time so this may delete a file while it is being used. Maybe we should move the generation of the file to be in `${cacheDir}/temp/${fileNameTmp}` so we don't need the clean-up.

Reply:
That's a good idea. I'll try to make some modifications.

Reply:
When I tried to move the file generation to the ${cacheDir} directory, I found that the value of cacheDir could not be obtained in the loadConfigFromBundledFile method. Therefore, I considered whether it was possible to import directly in base64 format without using a temporary file